addons/compact-messages/addon.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Compact Messages",
+  "description": "Makes the messages page compact and easy to read.",
```

Comment:
I think "more compact and easier to read" would be better.

Reply:
ok